### PR TITLE
Make methods  in GeoJson and WellKnownText static

### PR DESCRIPTION
**Pull request by iverase**
GeoJson and WellKnownText are utility classes to serialise / deserialise GeoJSON and WKT respectively. In order to use them, we need to instantiate an object with the parameters that are used while reading a geometry. This change makes all methods on the utility class static and adds the need parameters for reading geometries as method parameters. 

In addition, GeometryValidators are classes that can work pretty well under singleton pattern. Therefore we change as well StandardValidator and GeographyValidator to work under that pattern.

### CODE REVIEW

File: libs/geo/src/main/java/org/elasticsearch/geometry/utils/GeographyValidator.java [hunk 1]
```diff
@@ -49,10 +52,14 @@
 
     private final boolean ignoreZValue;
 
-    public GeographyValidator(boolean ignoreZValue) {
+    protected GeographyValidator(boolean ignoreZValue) {
         this.ignoreZValue = ignoreZValue;
     }
 
+    public static GeometryValidator get(boolean ignoreZValue) {
```
Comment (imotov) [inline libs/geo/src/main/java/org/elasticsearch/geometry/utils/GeographyValidator.java:59]: Maybe rename it to "for" so it is more clear that it's just a syntactic sugar for picking one of static instances

**Review by imotov — APPROVED** [review]
LGTM. Nice change!

File: libs/geo/src/main/java/org/elasticsearch/geometry/utils/GeographyValidator.java (hunk 1 above)
Comment (iverase) [reply libs/geo/src/main/java/org/elasticsearch/geometry/utils/GeographyValidator.java:59]: for is an illegal name, do you have a different suggestion? forValue?

Comment (iverase) [reply libs/geo/src/main/java/org/elasticsearch/geometry/utils/GeographyValidator.java]: I have renamed to #instance.

**Pull request merged**